Formula/php.rb

```diff
@@ -0,0 +1,412 @@
+class Php < Formula
+  desc "General-purpose scripting language"
+  homepage "https://php.net"
+  url "https://github.com/php/php-src/archive/php-7.1.7.tar.gz"
+  sha256 "fc57fde1df31e34fc6d58ea4ec477429d3663187391e2b307444b532dc18550d"
+
+  # javian: not sure about the origin of this so I'll keep it commented for now. Could it have something to do with building extensions ?
+  # So PHP extensions don't report missing symbols
+  #skip_clean "bin", "sbin"
+
+  # TODO
+  # Opcache default config file is included in extension formula, is it needed ?
+  # ldap extension: I have a vague recollection that this caused some issues that were resolved with exluding it. needs to be checked.
+  # According to php docs enchant requires aspell as a dep, should not be an issue since aspell is required.
+  # Need to add tests for all supported SAPIs
+  # Should we remove --with-pdo-dblib from formula ? (mssql support is long gone and as far as I can tell this is the 
+  #  only purpose and would also exclude a dep, freetds)
+  # How should the Formula handle the apache module ? Sierra (have not checked high sierra) can't support building it for the OS 
+  #   (without manually fiddling with link in the file system) with the bundles tools
+
+  option "with-debug-symbols", "Compile with debugging symbols"
+  option "with-freetds", "Build MSSQL-DB support extension"
+  option "with-httpd24", "Enable building of shared Apache 2.4 Handler module"
+  option "with-httpd22", "Enable building of shared Apache 2.2 Handler module"
+  option "with-imap-uw", "Build IMAP extension"
```

Comment:
Only that you need to put the right settings in your hosts file. See https://serverfault.com/questions/586141/php-cli-with-imap-5-second-startup-delay